app/sweepers/tag_sweeper.rb

```diff
@@ -1,6 +1,6 @@
 class TagSweeper < ActionController::Caching::Sweeper
```

Comment:
I noticed tag_sweeper is commented out in several places now that we've moved its contents into the model. Are we still using this file rather than the model in some places?